packages/contractkit/src/wrappers/Attestations.ts

```diff
@@ -254,7 +239,7 @@ export class AttestationsWrapper extends BaseWrapper<Attestations> {
             name: nameClaim ? nameClaim.name : undefined,
           }
         } catch (error) {
-          console.error(error)
+          console.info('Error getting attestation service URLs', error)
```

Comment:
Why not logging using the error level?

Reply:
Because this error happens quite often and it causes red banners

Reply:
ok but this is normal in this process, right?
Maybe just add a comment.

Reply:
Will do!